nginx-ingress-istio/ingress-ingress-ns.yaml

```diff
@@ -0,0 +1,14 @@
+# ingress-ingress-ns.yaml
```

Comment:
@ZackButcher Just to verify my understanding, since there's another Ingress resource in the default namespace, which nginx-ingress-controller will enforce this Ingress resource? Is it the nginx-ingress-controller in the same namespace, i.e. ingress namespace?